Jenkinsfile.github

```diff
@@ -0,0 +1,96 @@
+/*
+ * MIT License
+ *
+ * (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ * OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+@Library('csm-shared-library') _
+
+pipeline {
+    agent {
+        label 'metal-gcp-builder'
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '10'))
+        timestamps()
+    }
+
+    environment {
+        NAME = 'docs-sat'
+        IS_STABLE = getBuildIsStable(releaseBranchIsStable: true)
+        BUILD_METADATA = getRpmRevision(isStable: env.IS_STABLE)
+        RELEASE_BRANCH_VERSION = getReleaseBranchVersion()
+    }
+
+    stages {
+
+        stage('Build RPM Package') {
+            agent {
+                docker {
+                    image 'artifactory.algol60.net/csm-docker/stable/csm-docker-sle:15.3'
+                    reuseNode true
+                }
+            }
+            steps {
+                sh 'make prepare'
+                sh 'make rpm_package_source'
+                sh 'make rpm_build_source'
+                sh 'make rpm_build'
+            }
+        }
+
+        stage('Publish') {
+            steps {
+                script{
+                    def artifactory_component = env.NAME
+                    if (env.IS_STABLE.toBoolean() && env.RELEASE_BRANCH_VERSION) {
```

Comment:
The way that we create the `-latest.noarch.rpm` is mimicking CSM: https://github.com/Cray-HPE/docs-csm/blob/d93179604b12e6c2c53c474b63509fa847bd3389/Jenkinsfile.github#L97-L108

However I'm not creating a `-latest.src.rpm` as we wouldn't refer to it anywhere.